config/index.js

```diff
@@ -0,0 +1,2 @@
+const config = require('./env/default');

```

Comment:
`export const env = process.env.NODE_ENV;`

We have to add this variable also and then require config by `env`